lib/Services/SettingsService.php

```diff
@@ -178,6 +181,17 @@ public function getImageProvider($name): Provider {
     }
 
 
+    /**
+     * Get the selected imageprovider
+     *
+     * @return string current provider
```

Comment:
This doesn't return a string, but a `Provider`. It uses a string, maybe an Id would be better :)